modules/fb_instant_articles_rss/includes/rss.inc

```diff
@@ -46,3 +46,43 @@ function fb_instant_articles_rss_page() {
 
   echo $rss->asXML();
 }
+
```

Comment:
The `module_load_include` statement in `fb_instant_articles_rss_page` needs to change to:

```
module_load_include('inc', 'fb_instant_articles_rss', 'includes/FacebookInstantArticlesSimpleXMLElement');
`